src/app/dataset-view/dataset-view-menu/dataset-view-menu.component.html

```diff
@@ -217,13 +729,66 @@
+                                                        role="button"
+                                                    >
+                                                        <div style="position: relative; display: inline-block">
+                                                            <svg-icon name="copy"></svg-icon>
+                                                        </div>
+                                                        <div style="position: relative; display: none">
+                                                            <svg-icon name="checked"></svg-icon>
+                                                        </div>
+                                                    </button>
+                                                </div>
+                                            </div>
+                                        </div>
+
+                                        <div class="w-100">
+                                            <mat-divider class="my-2"></mat-divider>
+                                            <div class="d-flex align-items-center">
+                                                <mat-icon class="text-muted fs-5">info</mat-icon>
+                                                <span class="fs-12">
+                                                    <a
+                                                        href="https://docs.kamu.dev/node/protocols/odata/"
+                                                        target="_blank"
+                                                        class="fs-12"
+                                                        >OData adapter details</a
+                                                    >
+                                                </span>
+                                            </div>
+                                        </div>
                                     </div>
-                                    <button class="select-menu-item">As Parquet</button>
-                                    <button class="select-menu-item">As CSV</button>
                                 </mat-tab>
-                                <mat-tab label="Cite">
-                                    <button class="select-menu-item">DOI</button>
-                                    <button class="select-menu-item">Bibtech</button>
+                                <mat-tab>
+                                    <ng-template mat-tab-label>
+                                        <div class="d-flex align-items-center">
+                                            <mat-icon class="text-muted fs-5 mt-2">ios_share</mat-icon>
+                                            Export
+                                        </div>
+                                    </ng-template>
+                                    <button mat-menu-item class="select-menu-item" [disabled]="true">as Parquet</button>
```

Comment:
Let's replace format selection with a drop-down and add a "Dowload" button (disabled for now)

Reply:
Fixed 

![image](https://github.com/kamu-data/kamu-web-ui/assets/51016717/3549eec3-dad9-471a-94c1-eb3d0f659543)